php/Plugin.php

```diff
@@ -92,7 +93,13 @@ public function require_deprecated() {
 	 * Requires helper functions.
 	 */
 	private function require_helpers() {
-		require_once __DIR__ . '/Helpers.php';
 		require_once __DIR__ . '/BlockApi.php';
+
+		if ( function_exists( 'block_field' ) || function_exists( 'block_row' ) ) {
```

Comment:
```suggestion
		if ( function_exists( 'block_field' ) || function_exists( 'block_row' ) || function_exists( 'block_rows' ) ) {
```

Would this work?

Reply:
Ah, interesting

Reply:
That might do it

Reply:
It looks like a fix is needed in the Pro plugin: https://github.com/studiopress/genesis-custom-blocks-pro/pull/10